src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -301,6 +301,20 @@ public AzureAppConfigurationOptions Connect(IEnumerable<string> connectionString
             return this;
         }
 
+        /// <summary>
+        /// Connect the provider to CDN endpoint.
+        /// </summary>
+        /// <param name="cdnEndpoint">The endpoint of the CDN instance to connect to.</param>
+        public AzureAppConfigurationOptions ConnectCdn(Uri cdnEndpoint)
```

Comment:
We allow users to specify multiple `Connect` calls and the last one always wins. 
We are preserving that behavior with `ConnectCdn`, but with CDN endpoint involved, it's not that intuitive. I think we should throw if Connect and ConnectCdn are invoked together. This should not be allowed:

```
options.Connect(appConfigEndpoint, tokenCredential)
       .ConnectCdn(cdnEndpoint)
```

@zhenlan @drago-draganov what do you think?

Reply:
What is "not that intuitive" ?

Reply:
`Connect` is loading from AppConfig service, but `ConnectCdn` is loading from CDN service. With a setup like above, I would expect my provider to load data from both endpoints.

Reply:
Agree with Avani's suggestion - the Connect and ConnectCdn shouldn't be allowed to be used at the same time.

Even if there is such a scenario, users can create two separate configuration builders.

Reply:
Done